test/instrumentation/dropped-span-stats.js

```diff
@@ -0,0 +1,133 @@
+'use strict'
```

Comment:
This filename needs to be "*.test.js".

Reply:
9a2d1a3a02e7b6469f182b204af2f193fa7b1d48